.goreleaser.yml

```diff
@@ -15,37 +13,18 @@ builds:
       - -X keboola-as-code/src/build.BuildVersion={{ if index .Env "TARGET_VERSION"  }}{{ .Env.TARGET_VERSION }}{{ else }}dev{{ end }}
       - -X keboola-as-code/src/build.BuildDate={{.Date}}
       - -X keboola-as-code/src/build.GitCommit={{.Commit}}
-    ignore:
-      - goos: windows
-        goarch: arm64
+    hooks:
+      # Copy binary to TARGET_PATH if set
+      post: '{{ if index .Env "TARGET_PATH"  }}cp "{{ .Path }}" "{{ .Env.TARGET_PATH }}"{{ else }}true{{ end }}'
 
 changelog:
-  # Set it to true if you wish to skip the changelog generation.
-  # This may result in an empty release notes on GitHub/GitLab/Gitea.
   skip: true
 
 archives:
-  - format_overrides:
-      - goos: "windows"
-        format: "zip"
-    id: kbc
-    # dont package README and LICENSE files to the release archive
-    files:
+  - files:
       - none*
-    builds:
-      - kbc
-    name_template: "kbc-v{{ .Version }}-{{ .Os }}_{{ .Arch }}"
-    replacements:
```

Comment:
Myslim, ze toto nepotrebujeme, kedze rucna instalacia bude uz posledna moznost. Teda preferujem, nechat to co najjednoduchsie.